src/Controls.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import './controls.css';
+
+class Controls extends Component {
+  render() {
+    return (
+      <React.Fragment>
```

Comment:
```js
import React, { Component } from 'react';

// some code

return (
  <Fragment>
  </Fragment>
);
```

Do this instead of React.Fragment